src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -6835,6 +6839,35 @@ public override IOperation VisitSwitchExpression(ISwitchExpressionOperation oper
             return GetCaptureReference(captureOutput, operation);
         }
 
+        private void VisitUsingVariableDeclarationOperation(VariableDeclarationGroupOperation operation, ImmutableArray<IOperation> statements)
+        {
+            IOperation saveCurrentStatement = _currentStatement;
+            _currentStatement = operation;
+            StartVisitingStatement(operation);
+
+            var declarationKind = operation.DeclarationKind;
+            Debug.Assert(declarationKind == VariableDeclarationKind.Using || declarationKind == VariableDeclarationKind.AsynchronousUsing);
+
+            // a using statement introduces a 'logical' block after declaration, we synthesize one here in order to analyze it like a regular using 
+            BlockOperation logicalBlock = new BlockOperation(
+                operations: statements,
+                locals: default,
```

Comment:
Do we ever create blocks with default locals? Perhaps using an empty array would be more robust.